### DISCUSSION TITLE
How to enumerate/filter a subset of domains containing wildcards?

### DISCUSSION THREAD

**Discussion by vansh1** [projectdiscovery/subfinder#647] [Q&A] [answered]
how to enumerate multi-subdomains in middle of domain like
like support.*.hackeron.com

**Comment by forgedhallpass** [comment, marked as answer]
You have a couple of options to achieve this:
1. You do passive searching for all the subdomains, and then filter the results manually using `grep` (or using the freshly implemented https://github.com/projectdiscovery/subfinder/issues/575 feature, which is available on the `dev` branch)

```sh
subfinder -d hackerone.com -match "support*.hackerone.com" 
```

2. You create generate the subdomains using a wordlist and then use [`dnsx`](https://github.com/projectdiscovery/dnsx) to validate the results.

```sh
for word in $(cat wordlist.txt); do echo "support.$word.hackerone.com" >> generated.txt; done
cat generated.txt | dnsx
``` 

There is also a [feature request](https://github.com/projectdiscovery/dnsx/issues/155) in `dnsx`. You can subscribe to it in order receive a notification when it gets implemented.
